### DISCUSSION TITLE
How to access Isaaclab Nucleus server

### DISCUSSION THREAD

**Discussion by dxyy1** [isaac-sim/IsaacLab#591] [Q&A] [answered]
Hi there!

I noticed that all the pretrained model and the actuator nets are all loaded directly from the Isaac Nucleus server. I probability missed this in the documentation but does anyone know how we can 1) inspect the isaaclab server content from the Omniverse Nucleus app, 2) load existing models (.pt files) from the server, other than the ones existing in the Tutorials?

Apologies for the naive questions, and thanks in advance :).
![image](https://github.com/isaac-sim/IsaacLab/assets/139338590/e7709b40-4a2f-43b2-96af-56bd7a992b0e)

**Comment by Mayankm96** [comment]
This seems a general Isaac Sim related question. The instructions here should help: https://docs.omniverse.nvidia.com/isaacsim/latest/installation/install_faq.html?highlight=nucleus#assets-and-nucleus

**Reply by dxyy1** [reply]
Hi there,
Thank you for your reply! I want to access the Isaaclab nucleus server specifically, which I am not too sure if it is even publicly assessible to start with. This is because from the Tutorials, I saw that there are models being loaded from the Isaaclab Nucleus Server but no ntoes explaining the content of that server or how to access the other models, which is why I am asking here. 

Apologies if my previous question was not clear, and I hope this clears it up. :)

**Reply by Mayankm96** [reply, marked as answer]
Once you are able to connect to `Isaac` nucleus directory, you should find the folder `IsaacLab` in there.

Here's how it is organized in the code:
> https://isaac-sim.github.io/IsaacLab/source/api/lab/omni.isaac.lab.utils.html#omni.isaac.lab.utils.assets.ISAACLAB_NUCLEUS_DIR

**Reply by dxyy1** [reply]
Thank you very much!

**Reply by ihlej** [reply]
Hi, 
I am having difficulties understanding how this works, I have a related question here: https://github.com/isaac-sim/IsaacLab/discussions/890
Is it possible for me to inspect these assets visually via **<ip>/streaming/webrtc-demo/?server=<ip>**? (I work on a remote desktop using webrtc). I have managed to download a USD from the URL used in the anymal_c example code, 
http://omniverse-content-production.s3-us-west-2.amazonaws.com/Assets/Isaac/4.0/Isaac/IsaacLab/Robots/ANYbotics/ANYmal-C/anymal_c.usd
but it seems Props/instanceable_meshes.usd doesn't exist, so I am unable to see anything, which I would like to be able to do.

**Reply by Gariscat** [reply]
> Hi, I am having difficulties understanding how this works, I have a related question here: #890 Is it possible for me to inspect these assets visually via **/streaming/webrtc-demo/?server=**? (I work on a remote desktop using webrtc). I have managed to download a USD from the URL used in the anymal_c example code, http://omniverse-content-production.s3-us-west-2.amazonaws.com/Assets/Isaac/4.0/Isaac/IsaacLab/Robots/ANYbotics/ANYmal-C/anymal_c.usd but it seems Props/instanceable_meshes.usd doesn't exist, so I am unable to see anything, which I would like to be able to do.

Faced the same issue, for `http://omniverse-content-production.s3-us-west-2.amazonaws.com/Assets/Isaac/4.5/Isaac/Props/Sektion_Cabinet/sektion_cabinet_instanceable.usd`

**Comment by dxyy1** [comment]
Hi, I am not sure if you can see it through the server. But the way I have been viewing these assets is through an app call USD Composer that can be downloaded in the soon-to-be deprecated Omniverse Launcher.

Usd composer is an app you can find on the Exchange tab in the Launcher.

I hope this helps.


Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
